src/block/block_ckpt.c

```diff
@@ -90,23 +115,39 @@ __wt_block_checkpoint_load(WT_SESSION_IMPL *session, WT_BLOCK *block, const uint
         WT_ERR(__wt_block_ckpt_unpack(session, block, addr, addr_size, ci));
 
         /* Verify sets up next. */
-        if (block->verify)
+        if (block->verify) {
+            /* Verify isn't supported on tiered tables and should return before getting here. */
```

Comment:
Does this comment need any kind of marker such as FIXME or TODO?

Reply:
Added a FIXME. There isn't a WT ticket; it's a full project. So I don't cite a ticket number since PM tickets aren't publicly visible.